vici/session.go

```diff
@@ -45,12 +47,24 @@ type Session struct {
 
 // NewSession returns a new vici session.
 func NewSession(opts ...SessionOption) (*Session, error) {
+
+	var socketPath string
+	var network string
+	fmt.Println("os type: " + runtime.GOOS)
+	if runtime.GOOS == "windows" {
```

Comment:
As noted in my other comment, we should use build constraints instead of doing this at runtime.

Reply:
yes, your way is better, tks your suggestion.